flatpak/make_credentials_fetcher_flatpak.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
```

Comment:
Would be good to add a header saying what this script does and how to invoke it.